macros/edr/dbt_artifacts/upload_source_freshness_results.sql

```diff
@@ -0,0 +1,37 @@
+{% macro upload_source_freshness_results() %}
+  {% set source_freshness_results_relation = ref('source_freshness_results') %}
```

Comment:
Maybe for consistency call it `dbt_source_freshness`?

Reply:
But this is not an artifact, it's a results table.
It grows with time (incremental) and doesn't represent the latest state like the artifacts do.

Reply:
so maybe just add the prefix dbt (dbt_source_freshness_results)? like dbt_run_results?